packages/riipen-ui/src/components/Chip.test.jsx

```diff
@@ -0,0 +1,136 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Chip from "./Chip";
+
+it("renders without errors", () => {
+  try {
+    mount(<Chip />);
+  } catch (error) {
+    expect(true).toEqual(false);
```

Comment:
This won't actually work. If the chip does not throw an error, the `catch` block never actually triggers.

I think what you want is:

```js
let error;

try {
  mount(<Chip />);
} catch (e) {
  error = e;
}

expect(error).toEqual(undefined);
```

In the example above `expect(error).toEqual(undefined);` always runs and the error actually only gets set if the `catch` block triggers.

You could also do this same thing with a `try / catch / finally` as described here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch but that is probably overkill.